packages/wonder-blocks-core/util/get-clickable-behavior.test.js

```diff
@@ -0,0 +1,97 @@
+// @flow
+import * as React from "react";
+import {MemoryRouter} from "react-router-dom";
+import ClickableBehavior from "../components/clickable-behavior.js";
+import getClickableBehavior from "./get-clickable-behavior";
+
+describe("getClickableBehavior", () => {
+    test("Without href, returns ClickableBehavior", () => {
+        // Arrange
+        const url = undefined;
+        const clientNav = undefined;
+        const router = undefined;
+        const expectation = ClickableBehavior;
+
+        // Act
+        const result = getClickableBehavior(url, clientNav, router);
+
+        // Assert
+        expect(result).toBe(expectation);
+    });
+
+    describe("with href", () => {
+        test("External URL, returns ClickableBehavior", () => {
+            // Arrange
+            const url = "http://google.com";
+            const clientNav = undefined;
+            const router = <MemoryRouter />;
+            const expectation = ClickableBehavior;
+
+            // Act
+            const result = getClickableBehavior(url, clientNav, router);
+
+            // Assert
+            expect(result).toBe(expectation);
+        });
+
+        describe("Internal URL", () => {
+            describe("Client navigation is undefined", () => {
```

Comment:
Why differentiate between `clientNav` undefined vs. false?

Reply:
Because the code does. When `clientNav` isn't specified, it defers to the existence of a `router`. When it's specified, it ignores the router and makes a decision base don the `clientNav` value.

IF that's not the behaviour we want, then the method will need updating. I just captured its current behaviour.

Reply:
@somewhatabstract I didn't think that through carefully.  I think we still want to provide a way to create links that aren't client-side nav links when the router is present.  Although we can do that by explicitly setting `clientNav={false}` it's confusing b/c often with booleans not setting them is equivalent to explicitly setting them to false.  I also wonder if we should make `clientNav` mandatory even when the router is present otherwise people might be confused as to why they're seeing client side navigation when `clientNav` isn't present.  @jeresig what do you think.